proknow/Patients/Entities/ImageSets.py

```diff
@@ -68,3 +68,37 @@ def download(self, path):
             image_path = os.path.join(main_directory, modality + "." + image["uid"])
             self._requestor.stream('/workspaces/' + self._workspace_id + '/imagesets/' + self._id + '/images/' + image["id"] + '/dicom', image_path)
         return main_directory
+
+    def get_slice_data(self, index):
```

Comment:
Do you think this method should be called `get_image_data` since it is technically getting data associated with an item from the `"images"` array (as opposed to dose, which is actually getting it from the `"slices"` array). An alternative method name that would be consistent for both would be `get_pixel_data`. Though I think I'm leaning towards `get_image_data` and `get_slice_data`.

Reply:
I'm fine with `get_image_data`.